shared/middleware/helpers/core.rb

```diff
@@ -46,9 +46,19 @@ def not_found()
 end
 
 def too_large()
-  halt(413, "Payload too large\n")
```

Comment:
I think you are correct Laurel -- the helpers in this file only work when called from within a class that inherits from Sinatra::Base.